.github/workflows/danger.yml

```diff
@@ -32,10 +34,14 @@ jobs:
         # frozen-lockfile will make the build fail if the lockfile is not there
         run: yarn install --frozen-lockfile
 
-      - name: Check
+      - name: Validate Labels
+        run: |
+          yarn run danger ci \
+            --dangerfile Automattic/peril-settings/org/pr/label.ts \
+            --id pr_labels
```

Comment:
The `--id` option will make sure that no Danger comment will be lost. Without it, if two or more `danger ci` runs have to post a comment, the last one doing it will delete the content from the previous ones.